Neos.Neos/Resources/Private/Translations/de/Main.xlf

```diff
@@ -465,10 +465,10 @@
 			<target xml:lang="de">Eine gültige Zahl wird erwartet.</target></trans-unit>
       <trans-unit id="content.inspector.validators.numberRangeValidator.numberShouldBeInRange" xml:space="preserve" approved="yes">
 				<source>Please enter a valid number between {minimum} and {maximum}</source>
-			<target xml:lang="de">Bitte geben Sie eine gültige Zahl zwischen {{minimum}} und {{maximum}} ein.</target><alt-trans><target xml:lang="de">Bitte geben Sie eine gültige Zahl zwischen {minimum} und {maximum} ein.</target></alt-trans></trans-unit>
+			<target xml:lang="de">Bitte geben Sie eine gültige Zahl zwischen {minimum} und {maximum} ein.</target><alt-trans><target xml:lang="de">Bitte geben Sie eine gültige Zahl zwischen {minimum} und {maximum} ein.</target></alt-trans></trans-unit>
       <trans-unit id="content.inspector.validators.regularExpressionValidator.patternDoesNotMatch" xml:space="preserve" approved="yes">
 				<source>The given subject did not match the pattern ({pattern})</source>
-			<target xml:lang="de">Der angegebene Wert stimmt nicht mit dem Muster ({{pattern}}) überein</target><alt-trans><target xml:lang="de">Der angegebene Wert stimmt nicht mit dem Muster ({pattern}) überein</target></alt-trans><alt-trans><target xml:lang="de">Das gegebene Thema stimmt nicht mit dem Muster ({{pattern}}) überein</target></alt-trans></trans-unit>
+			<target xml:lang="de">Der angegebene Wert stimmt nicht mit dem Muster ({pattern}) überein</target><alt-trans><target xml:lang="de">Der angegebene Wert stimmt nicht mit dem Muster ({pattern}) überein</target></alt-trans><alt-trans><target xml:lang="de">Das gegebene Thema stimmt nicht mit dem Muster ({pattern}) überein</target></alt-trans></trans-unit>
       <trans-unit id="content.inspector.validators.stringValidator.stringIsExpected" xml:space="preserve" approved="yes">
 				<source>A valid string is expected.</source>
 			<target xml:lang="de">Eine gültige Zeichenfolge wird erwartet.</target></trans-unit>
```

Comment:
hey you fixed this in the German translation file instead of the English one which is default, meaning this will be overridden automatically by our automated jobs.. to fix this it needs to be fixed in all translations files or just in the english default one

Reply:
Hi, Thank you for your comment first of all but as you can see this is my first commit and I have not figured out exactly how the Translation Process works in Neos. I guess I have figured it out where is the problem. (I guess here: https://github.com/neos/neos-ui/blob/master/packages/neos-ui-validators/src/StringLength/index.js#L29 ) but I do not know how can I locally run the automation job to test my changes locally before committing them.

is there a documentation somewhere about the whole process? and more specifically how these files are produced?

Thanks again

Reply:
@amirkoklan Welcome and congratulations to your first Neos PR!
@aertmann what do you mean, it has to be fixed in the english default one? The typo is not in there ;)

Amir, the translation process is documented here: https://www.neos.io/contribute/translating-neos.html

I found 119 occurrences of "{{" in the Neos translation files spread over 14 files (in 2.3). Maybe we can fix them all at once

Reply:
Perfect! Thank You. 
I'll take care of them all at once.

Reply:
The german labels were already fixed by @jonnitto, I just approved those.

Reply:
@bwaidelich: if it's only in the German translations, then it will probably be overridden in the next export if fixed manually in the non-default translations files.. so it probably needs to be fixed on translate.neos.io instead /cc @amirkoklan